server/src/main/java/com/defold/extender/builders/CSharpBuilder.java

```diff
@@ -197,7 +203,42 @@ private static ArrayList<String> makePathsAbsolute(String basePath, ArrayList<St
         return out;
     }
 
-    private static void addLibFlags(String platform, List<String> linkFlags) throws IOException {
+    private static void getExportsFlags(String platform, File buildDir, List<String> linkFlags) throws IOException {
+        // We need to add the exported symbols map
+        File exportsFile = new File(String.format("%s/defold_cs.exports", buildDir));
+
+        String exportsPattern = "-Wl,--version-script=%s";
+        if (ExtenderUtil.isMacOSTarget(platform) || ExtenderUtil.isIOSTarget(platform))
+        {
+            exportsPattern = "-exported_symbols_list \"%s\"";
+        }
+
+        // HACK: In anticipation of the fix for https://github.com/dotnet/runtime/issues/109341
+        // we have to make sure not all symbols are public, and at the same time respect the engine symbols
+        String contents = "";
+        contents += "V1.0 { \n";
```

Comment:
Suggestion: create a resource file (the same as we do for podfile.template) and read it instead of building string every time. It can be useful in the future if exports list will become a template.

Reply:
Yeah, this is still just a hack, until I can get hold of Dotnet 10, where this should be fixed. So I opted out of creating the template file etc.